runtime/observability/telemetry-api/common/src/aws/smithy/kotlin/runtime/telemetry/context/AbstractContext.kt

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package aws.smithy.kotlin.runtime.telemetry.context
+
+/**
+ * An abstract implementation of a telemetry provider. By default, this class uses no-op implementations for all members
```

Comment:
docs: "An abstract implementation of a telemetry **provider's context**"?

Reply:
Yep, hasty copy/paste job. Good catch.